addons/udes_security/controllers/authSignupHome.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Filenames should be all lowercase, with underscore separators e.g. `auth_signup_home.py`
Again this is a python style guidelines: https://www.python.org/dev/peps/pep-0008/#package-and-module-names